src/clitools/parser.py

```diff
@@ -11,29 +11,49 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import sys
+from typing import List
+
 import click
 
-from src.model.spdx_no_assertion import SpdxNoAssertion
-from src.model.spdx_none import SpdxNone
+from src.model.document import Document
+from src.parser.parse_anything import parse_file
+from src.validation.document_validator import validate_full_spdx_document
+from src.validation.validation_message import ValidationMessage
+from src.writer.tagvalue.tagvalue_writer import write_document
 
 
 @click.command()
 @click.option("--file", prompt="File name", help="The file to be parsed")
-@click.option("--force", is_flag=True, help="print information even if there are some parsing errors")
-def main(file, force):
+@click.option("--version", prompt="SPDX version", help="The SPDX version to be used during validation")
+@click.option("--validate", is_flag=True, help="validate the provided document")
+@click.option("--printout", is_flag=True, help="print the parsed document to stdout in tag-value format")
+def main(file, version, validate, printout):
```

Comment:
why is there a parser and a converter? I think they both could be the same tool.